docs/destinations/forwarding-events/snowbridge/3-X-X-upgrade-guide/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "3.X.X Upgrade Guide"
+date: "2022-10-20"
+sidebar_position: 400
+---
+
+## Breaking Changes
+
+The below breaking changes were made in version 3.0.0. All other functionality is backwards compatible.
+
+### Lua support removed
+
+Support for Lua transformations has been removed. If you are running a Lua transformation, you can port the logic to [Javascript](/docs/destinations/forwarding-events/snowbridge/configuration/transformations/custom-scripts/javascript-configuration/index.md), or [JQ](/docs/destinations/forwarding-events/snowbridge/configuration/transformations/builtin/jq.md).
+
+### HTTP Target Changes
+
+Previously, by default the body of http requests came in whatever form it received. It is now an array. From v3, where no template is configured, the POST request body will contain an array of JSON containing the data for the whole batch. Data must be valid JSON or it will be considered invalid and sent to the failure target.
```

Comment:
Is there a reason we can’t describe these two changes separately instead of as “one change breaking in 2 ways”? I think that would be clearer. My mind is fried trying to track how the two aspects are related (I am not saying they are not!)

Reply:
What are the two changes you're referring to here?